### PR TITLE
also run easyconfigs tests with Python 3.6

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,9 @@ matrix:
     # also test default configuration with Python 2.7
     - python: 2.7
       env: LMOD_VERSION=6.6.3
+    # also test with Python 3.6
+    - python: 3.6
```
Comment (akesandgren) [inline .travis.yml]: Doesn't this depend on testing branch 4.x?

Comment (boegel) [reply .travis.yml]: It does, the Travis config was changed to build against the `4.x` branch of of `easybuild-framework` and `easybuild-easyblocks` when the target branch for the PR is `4.x`, so it's all good?

Comment (akesandgren) [reply .travis.yml]: Ah, yes... this is of course the 4.x branch itself ...
This week has not been good for brain...

**Comment by vanzod** [comment]
Re-trigger Travis

**Comment by akesandgren** [comment]
Rekicking travis again.

**Comment by akesandgren** [comment]
Travis kicked

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**